### PR TITLE
Atcoder site API integration completed.

### PR DESCRIPTION
**Pull request by abdealijaroli**
API integration for Codeforces, CodeChef, and Atcoder websites are completed now. The bot is fully functional for the mentioned sites!

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
   codeforcesErr: 'Oops, looks like there are no upcoming contests currently on Codeforces.',
   codeforcesReply: 'Upcoming contests on Codeforces are - ',
   codeChefUrl: 'http://cp-events-api.herokuapp.com/api/v1/codechef',
-  codeChefErr: 'Oops, looks like there are no upcoming contests currently on Codechef.',
-  codeChefReply: 'Upcoming contests on CodeChef are - '
+  codeChefErr: 'Oops, looks like there are no upcoming contests currently on CodeChef.',
+  codeChefReply: 'Upcoming contests on CodeChef are - ',
+  atcoderUrl: 'http://cp-events-api.herokuapp.com/api/v1/atcoder',
+  atcoderErr: 'Oops, looks like there are no upcoming contests currently on Atcoder.',
+  atcoderReply: 'Upcoming contests on Atcoder are - '
```
Comment (mdhishaamakhtar) [inline src/constants.ts]: use camelCase here too

File: src/constants.ts [hunk 2]
```diff
@@ -1,10 +1,13 @@
 export const constants = {
   startMessage: "Hello Coder, great to have you on board! Let's get started: type /help to get the list of all the available websites.",
-  helpMessage: 'Type the number corresponding to the site to get the upcoming contest details for the same. \n\n1. Codeforces \n2. CodeChef \n3. Topcoder \n4. Atcoder \n5. ICPC \n6. Google Kick Start',
+  helpMessage: 'Type the number corresponding to the site to get the upcoming contest details for the same. \n\n1. Codeforces \n2. CodeChef \n3. Atcoder \n4. Topcoder \n5. ICPC \n6. Google Kick Start',
   codeforcesUrl: 'https://codeforces.com/api/contest.list',
   codeforcesErr: 'Oops, looks like there are no upcoming contests currently on Codeforces.',
   codeforcesReply: 'Upcoming contests on Codeforces are - ',
   codeChefUrl: 'http://cp-events-api.herokuapp.com/api/v1/codechef',
```
Comment (mdhishaamakhtar) [inline src/constants.ts]: use camelCase here

**Review by mdhishaamakhtar — CHANGES_REQUESTED** [review]

**Review by mdhishaamakhtar — COMMENTED** [review]
Make them camelCase and I'll merge

**Comment by abdealijaroli** [comment]
Requested changes done

**Review by mdhishaamakhtar — APPROVED** [review]

**Pull request merged**